src/luarocks/build.lua

```diff
@@ -12,6 +12,7 @@ local cfg = require("luarocks.core.cfg")
 local repos = require("luarocks.repos")
 local writer = require("luarocks.manif.writer")
 local deplocks = require("luarocks.deplocks")
+local vers = require("luarocks.core.vers")
```

Comment:
nit: could you move this up next to the other `core` module required?

Reply:
Alright!